build.sh

```diff
@@ -3,8 +3,8 @@
 set -eo pipefail
 
 readonly CNAME_ADDRESS="standards.oftrust.net"
-readonly WORKDIR="/src"
-readonly ONT_FILE="/src/ontologies/pot.jsonld"
+readonly WORKDIR="/src/v1"
```

Comment:
You can't just discard `v1` when deploying the `Context`, `Vocabulary` and `ClassDefinitions` -files.

You probably also should not do that for the files generated by `ontospy`

The context etc files must end up in `https://standards.oftrust.net/v1/Context/...` and I would find it really weird if we only had one HTML generated which is some random version of the standards instead of the `v1`/`v1.1`/`v2` that you're actually interested in

Reply:
https://help.github.com/en/articles/redirects-on-github-pages We'll likely want to set up a redirect like this from `/index.html` that ends up in the GitHub pages that then points to `/v1/index.html`

Reply:
Added Jekyll configuration